config/default/kustomization.yaml

```diff
@@ -1,5 +1,5 @@
 # Adds namespace to all resources.
-namespace: keda
+#namespace: keda
```

Comment:
Comment or delete?

Reply:
Yeah, I don't like commented code as well, but in this case I'd like to keep it there. Because in the whole file, there are commented sections of examples of configuration for the Kustomization.